packages/thirdweb/src/extensions/erc1155/write/mintToBatch.ts

```diff
@@ -0,0 +1,56 @@
+import { maxUint256 } from "viem";
+import { multicall } from "../../../extensions/common/__generated__/IMulticall/write/multicall.js";
+import { upload } from "../../../storage/upload.js";
+import type {
+  BaseTransactionOptions,
+  WithOverrides,
+} from "../../../transaction/types.js";
+import type { NFTInput } from "../../../utils/nft/parseNft.js";
+import { encodeMintTo } from "../__generated__/IMintableERC1155/write/mintTo.js";
+
+export type MintToBatchParams = WithOverrides<{
+  to: string;
+  nfts: Array<{
+    supply: bigint;
+    metadata: NFTInput | string;
+  }>;
+}>;
+
+export function mintToBatch(
```

Comment:
add TS doc

also make sure you add a comment about the size of the batch - should not be too big otherwise it will not fit / not be included in a block

Reply:
ty ser. was waiting for a review on the code before I add the docs, seems all good!